config/application.rb

```diff
@@ -215,6 +221,15 @@ class Application < Rails::Application
 
     # Use the database for sessions instead of the cookie-based default
     config.session_store :active_record_store, :secure => !!SETTINGS[:require_ssl]
+
+    def dynflow
+      return @dynflow if @dynflow.present?
+      @dynflow = defined?(ForemanTasks) ? ForemanTasks.dynflow :
+        Dynflow::Rails.new(Foreman::Dynflow::Configuration.new)
```

Comment:
nitpicl: I know I originally asked for it,  but if its multi line, then it can probably go back to the older version or 
```ruby
@dynflow = 
if defined(ForemanTask?)
  ForemanTasks.dynflow
else
  Dynflow::Rails.new(Foreman::Dynflow::Configuration.new)
end
```